src/aiohomeconnect/client.py

```diff
@@ -644,12 +672,20 @@ async def put_command(
             data=put_command.to_dict(),
         )
 
-    async def get_all_events(
+    async def get_events(
         self,
+        ha_id: str,
         *,
         accept_language: Language | None = None,
     ) -> ArrayOfEvents:
-        """Get stream of events for all appliances - NOT WORKING WITH SWAGGER.
+        """Get a list of events for one appliance.
+
+        If you want to do a one-time query of the current status, you can ask for
+        the content-type `application/vnd.bsh.sdk.v1+json` and get the status
+        as normal HTTP response.
+
+        If you want an ongoing stream of events in real time, ask for the content
+        type `text/event-stream` and you'll get a stream as Server Sent Events.
```

Comment:
This isn't applicable anymore I think since we removed the `ContentType` model and hardcoded the `Accept` header.

Do you think we can remove this method or do we need to keep it?

Reply:
That is a nice question. By calling this, you can get the events (event type EVENT) from an appliance. But they are sent through streaming function.
So in my opinion is this is no longer necessary. Is it okey if I delete it?

Reply:
Yes, let's delete it.